lib/my_zipcode_gem.rb

```diff
@@ -1,3 +1,4 @@
+require 'rails/generators/actions'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.